contracts/ISSVNetwork.sol

```diff
@@ -167,7 +185,8 @@ interface ISSVNetwork is ISSVNetworkCore {
      */
     function registerOperator(
         bytes calldata publicKey,
-        uint256 fee
+        uint256 fee,
+        address whitelisted
```

Comment:
This means that all users will have to put 0x00000...?  They cant just put 0 right?  If so its a bit of overhead for the average user who wants to register a non private operator.

Reply:
Thats great i wanted to request that just to save gas for the normal register operators not needing to pay for the extra param / checks!